utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/ethtool.lua

```diff
@@ -0,0 +1,83 @@
+local ethtool = require "ethtool"
```

Comment:
Thanks for reviewing and it. I'll make a test build somewhere next week, but I suspect that requiring `ethtool` from a file named `ethtool` will try to import itself. IIRC, that was the reason why I named the collector `ethtool-stats`.

Reply:
it works in my tests
```
node_ethtool_transmitted_queue_0_packets{device="eth2"} 11
node_scrape_collector_duration_seconds{collector="ethtool"} 0.00029516220092773
node_scrape_collector_success{collector="ethtool"} 1
```